core/src/main/java/com/scalar/db/util/ProtoUtils.java

```diff
@@ -74,18 +73,95 @@ public static com.scalar.db.rpc.Get toGet(Get get) {
 
   private static Key toKey(com.scalar.db.rpc.Key key, TableMetadata metadata) {
     Key.Builder builder = Key.newBuilder();
-    key.getValueList().forEach(v -> builder.add(toColumn(v.getName(), v, metadata)));
+
+    // For backward compatibility
+    if (!key.getValueList().isEmpty()) {
+      key.getValueList().forEach(v -> builder.add(toColumn(v.getName(), v)));
+    } else {
+      key.getColumnList().forEach(c -> builder.add(toColumn(c, metadata)));
+    }
```

Comment:
If `Value` in ProtoBuf is used in a request, that indicates an old client sends the request. We need to handle this situation for backward compatibility.